R/create_single_dose_dataset.R

```diff
@@ -0,0 +1,272 @@
+#' Pre-Defined Dose Frequencies
+#'
+#' These pre-defined dose frequencies are sourced from
+#' [CDISC](https://evs.nci.nih.gov/ftp1/CDISC/SDTM/SDTM%20Terminology.pdf). The
+#' number of rows to generate using `create_single_dose_dataset()` arguments
+#' `start_date` and `end_date` is derived from `DFFactor` and `DFUnit` with
+#' appropriate functions from `lubridate`.
+#'
+#' @details
+#' To see the entire table in the console, run `print(dose_freq_lookup)`.
+#'
+#' @seealso [create_single_dose_dataset()]
+#'
+#' @export
+#'
+#' @keywords dose_freq
```

Comment:
The `dose_freq` keyword is not used on the reference page, i.e., the documentation can not be reached on the website. I would propose to use `@keywords metadata` and add a corresponding subsection to the "Other Keywords" section of the reference page.

Reply:
Please update `_pkgdown.yml` to add a "Metadata" section to the reference page.